tests/bdd/requirements.txt

```diff
@@ -0,0 +1,12 @@
+asyncio
```

Comment:
what is the relation between what's here and what's in the nix-shell?

Reply:
These are the dependencies of the pytests. I've actually removed a bunch of stuff that wasn't required.